catchpoint/Makefile

```diff
@@ -0,0 +1,71 @@
+VM_NAME := catchpoint
+K8S_NAMESPACE := default
+CONFIG_FILE := jinja/variables/cloud-init.yaml
+TEMPLATE_DIR := jinja/templates
+OUTPUT_DIR := generated_configs
+GRAFANA_FLOW_VALUES := $(OUTPUT_DIR)/grafana-flow-k8s-template.yaml
+
+.PHONY: run-catchpoint-multipass
+run-catchpoint-multipass: render-config
+	@multipass launch -n $(VM_NAME) --memory 14G --disk 10G --cpus 4
```

Comment:
Does it really need 14 GB of memory?

Reply:
I currently have 1.5 GiB of memory with an additional 5.4 GiB in buffer/cache, so I tried to decrease that down to 8G. I however could not start the kubernetes because I got an error `❌  Exiting due to RSRC_OVER_ALLOC_MEM: Requested memory allocation 12000MB is more than your system limit 7920MB.`.  I think that explains why it was set up so high. Also got an error when cpus was smaller than 4.

Reply:
Interesting. Not sure it makes sense to have such a high amount of requested memory, but that's likely coming externally. For now let's just accept it and then revisit in the future if it's a problem.

That OK with you @mshahzeb ?